code/modules/client/preferences_savefile.dm

```diff
@@ -258,41 +245,22 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 		if (preference_type in value_cache)
 			write_preference(preference, preference.serialize(value_cache[preference_type]))
 
-	//general preferences
-	WRITE_FILE(S["lastchangelog"], lastchangelog)
```

Comment:
How are all these saved now? Mutable list?

Reply:
I think I forgot these actually

Reply:
oh wait no, this is for saving

Reply:
I originally was going to these reference a list, but then realized it was a dumb idea

Reply:
So wait I'm confused what's the answer to my question

Reply:
Oh you fixed it